service/worker/scheduler/spec.go

```diff
@@ -255,11 +256,15 @@ func validateInterval(i *schedpb.IntervalSpec) error {
 	if i == nil {
 		return errors.New("interval is nil")
 	}
+	if err := timestamp.ValidateProtoDuration(i.Interval); err != nil {
```

Comment:
Technically this can cause a nondeterminism error in schedule workflows, if someone is using a duration that was valid before and is invalid now. I would just leave out the changes to this file and we can add it back later after we're using the state machine impl.

Reply:
Removed those changes and added a TODO so we don't forget.